gems/spree/CVE-2013-1656.yml

```diff
@@ -0,0 +1,25 @@
+---
+gem: spree
+cve: 2013-1656
+url: https://blog.convisoappsec.com/en/spree-commerce-multiple-unsafe-reflection-vulnerabilities-cve-2013-1656
+title: |
+  Spree controller Parameter Arbitrary Ruby
+  Object Instantiation Command Execution
+
+  osvdb-92116: parameter="promotion_action"; controller="promotion_actions_controller.rb"
+  osvdb-91117: parameter="payment_method";   controller="payment_methods_controller.rb"
+  osvdb-91218: parameter="promotion";        controller="calculator_types_controller.rb"
+  osvdb-91219: parameter="promotion_rule";   controller="promotion_rules_controller.rb"
```

Comment:
`title` is supposed to be a single sentence. Also, I think it's acceptable to remove mention of OSVDB IDs.